src/erlang_ls_completion_provider.erl

```diff
@@ -43,12 +42,12 @@ handle_request({completion, Params}, State) ->
       TriggerCharacter = maps:get(<<"triggerCharacter">>, Context, <<>>),
       %% We subtract 1 to strip the character that triggered the
       %% completion from the string.
-      Length = case Character > 0 of
-                 true  -> 1;
-                 false -> 0
-               end,
+      Length = case Character > 0 of true -> 1; false -> 0 end,
       Prefix = erlang_ls_text:line(Text, Line, Character - Length),
-      {find_completion(Prefix, TriggerKind, TriggerCharacter), State};
+      Opts   = #{ trigger  => TriggerCharacter
```

Comment:
In general, I prefer to be explicit on the arguments passed (i.e. use the actual parameters instead of passing generic map). The reason is that we can then use Dialyzer to spot type errors.